pkg/web/apps/apps_psql_repository_test.go

```diff
@@ -1,56 +1,241 @@
-// +build integration
-
 package apps
 
 import (
-	"reflect"
 	"testing"
 
-	"github.com/aerogear/mobile-security-service/pkg/config"
-	"github.com/aerogear/mobile-security-service/pkg/db"
-
 	"github.com/aerogear/mobile-security-service/pkg/models"
+	sqlmock "gopkg.in/DATA-DOG/go-sqlmock.v1"
 )
 
-func Test_appsPostgreSQLRepository_GetApps(t *testing.T) {
-	c := config.Get()
-	dbConn, err := db.Connect(c.DB.ConnectionString, c.DB.MaxConnections)
+func Test_appsPostgreSQLRepository_GetAppsWillReturnTwoApps(t *testing.T) {
```

Comment:
Is it(Test_appsPostgreSQLRepository_GetAppsWillReturnTwoApps) a really good name? 
Shows that we are not following here thee go convention. See [here](https://golang.org/pkg/testing/#pkg-overview). I understand that it could be something as TestGetAllAppsFromRespository. Wdyt?

Reply:
The VS Code extension generates unit test names like this. I appended the function name with `ShouldReturnNoApps` because it describes what the function is expected to do. I can't from the link you provided where it says above is bad?

> Multiple example functions for a package/type/function/method may be provided by appending a distinct suffix to the name. The suffix must start with a lower-case letter.

Reply:
It was before `Test_appsPostgreSQLRepository_GetAppsWillReturnTwoApps` and not it is `Test_appsPostgreSQLRepository_GetApps` which indeed shows better. 👍 You can consider it solved.